test/basic-querying.test.js

```diff
@@ -577,6 +577,21 @@ describe('basic-querying', function() {
       });
     });
 
+    it('should query by using database operator', function(done) {
+      User.findOne({
+        where: {
+          role: {
+            '$exists': false,
+          },
+        },
+        order: 'order DESC',
+      }, function(err, user) {
+        should.not.exist(err);
+        should.not.exist(user);
```

Comment:
IMO, it's enough to check for user's id, possibly one more property. No need to check all user properties.